src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
https://angular.io/docs/ts/latest/api/forms/index/DefaultValueAccessor-directive.html
Maybe this'll clear it up a bit